src/applications/disability-benefits/526EZ/components/FormStartControls.jsx

```diff
@@ -7,23 +7,27 @@ import SaveInProgressIntro from '../../../../platform/forms/save-in-progress/Sav
 import { ITFErrorAlert, VerifiedAlert, UnauthenticatedAlert, UnverifiedAlert, ITFDescription } from '../helpers';
 
 export default function FormStartControls(props) {
-  const { user } = props;
+  const { user: { login: { currentlyLoggedIn }, profile: { verified, services } } } = props;
 
   const somethingWentWrong = props.ITFStatus && props.ITFStatus === 'expired' || // This may not be possible
                              props.errors && props.errors.length;
+  const hasEVSSClaimsService = !!(services.find((service) => service === 'evss-claims'));
+  const gateUnauthenticatedUser = !currentlyLoggedIn;
```

Comment:
Do you think we need this, vs. just using `{!currentlyLoggedIn}` directly within the `return (...)`? To me, 'no logged in' is easier to reason about than the extra layer of abstraction. Curious to hear other opinions.

Reply:
Sounds good, updated.